backend/src/routes/.eslintrc.js

```diff
@@ -1,24 +0,0 @@
-module.exports = {
```

Comment:
is this eslintrc anywhere else in the project? If not please put it back in but in a more appropriate location